src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -130,7 +131,8 @@ case class ClusterInitValues(googleProject: GoogleProject,
                              jupyterServerName: String,
                              proxyServerName: String,
                              jupyterInstallExtensionScript: String,
-                             jupyterExtensionUri: GoogleBucketUri)
+                             jupyterExtensionUri: GoogleBucketUri,
+                             userServiceAccountCredentials: String)
```

Comment:
Since this is a bucket URI, maybe make the type `GoogleBucketUri` and name this `userServiceAccountCredentialsUri`.

(I know `GoogleBucketUri` is just an alias for `String` now but hopefully we'll have richer types soon.)